client/my-sites/upgrades/checkout-thank-you/purchase-detail.jsx

```diff
@@ -42,7 +42,10 @@ const PurchaseDetail = ( {
 PurchaseDetail.propTypes = {
 	additionalClass: React.PropTypes.string,
 	buttonText: React.PropTypes.string,
-	description: React.PropTypes.string,
+	description: React.PropTypes.oneOfType( [
+		React.PropTypes.array,
```

Comment:
Is this something required by the changes introduced in this pull request? If that's the case I don't see why we would expect `description` to be an array but I'm probably missing something.